src/Components/Artwork/FillwidthItem.tsx

```diff
@@ -40,13 +43,38 @@ export interface FillwidthItemContainerProps
 }
 
 export class FillwidthItemContainer extends React.Component<
-  FillwidthItemContainerProps,
-  null
+  FillwidthItemContainerProps
 > {
   static defaultProps = {
     useRelay: true,
   }
 
+  getImageUrl() {
+    const imageURL = this.props.artwork.image.url
+
+    if (!imageURL) {
+      return null
+    }
+
+    const {
+      artwork: {
+        image: { aspect_ratio },
+      },
+    } = this.props
+
+    // Either scale or crop, based on if an aspect ratio is available.
+    const type = aspect_ratio ? "fit" : "fill"
+    const height = this.props.imageHeight * window.devicePixelRatio
+    const width = Math.floor(height * aspect_ratio * window.devicePixelRatio)
```

Comment:
Since the images are already small in dimension (they're still small in size).

Reply:
100 vs 200px seems all good!